tests/pages/regions/sticky_promo.py

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.base import BaseRegion
+
+
+class StickyPromo(BaseRegion):
+
+    _root_locator = (By.CLASS_NAME, 'mzp-c-sticky-promo')
+    _close_locator = (By.CLASS_NAME, 'mzp-c-sticky-promo-close')
+
+    def close(self):
+        promo = self.selenium.find_element(*self._root_locator)
+        self.find_element(*self._close_locator).click()
+        self.wait.until(lambda s: 'mzp-a-fade-out' in promo.get_attribute('class'))
```

Comment:
This should fix the [test failure](https://gitlab.com/mozmeao/www-config/-/jobs/841874461) in Chrome:

```suggestion
        self.wait.until(lambda s: not self.is_displayed)
```